### PR TITLE
Planning poker commands

### PR DESCRIPTION
**Pull request by lukeledet**
Also formats some other unrelated files.

This should have feature parity with the current poker commands except maybe next but that doesn't seem to work right now anyway.

### CODE REVIEW

File: lib/slax/poker.ex [hunk 1]
```diff
@@ -0,0 +1,76 @@
...
+      channel: channel_name,
+      issue: repo_and_issue
+    })
+    |> Repo.insert()
+
+    labels = Enum.map_join(issue["labels"], ", ", & &1["name"])
+    pr = if Map.has_key?(issue, "pull_request"), do: "(PR) "
+
+    response = """
+      Planning poker for #{repo_and_issue}.
```
Comment (Dacello) [inline lib/slax/poker.ex:19]: but what about i18n? This should use gettext for all of our spanish speaking employees no? :trollface:

File: lib/slax/poker/estimates.ex [hunk 2]
```diff
@@ -0,0 +1,24 @@
+defmodule Slax.Poker.Estimates do
+  use Slax.Context
+  alias Slax.Poker.Estimate
+
+  @valid_estimates [1, 2, 3, 5, 8, 13, 666]
```
Comment (Dacello) [inline lib/slax/poker/estimates.ex:5]: lol at 666

File: lib/slax/slack.ex [hunk 3]
```diff
@@ -65,7 +65,7 @@ defmodule Slax.Slack do
   Sends a message to slack with the given url
   """
   def send_message(url, message) do
-    request = Jason.encode!(message)
+    request = "{\"text\": \"#{message}\"}"
```
Comment (oohnoitz) [inline lib/slax/slack.ex:68]: Not sure what `message` contains, but might be possible for it to escape by accident i think?

**Review by Dacello — DISMISSED** [review]
seems reasonable to me

File: lib/slax/slack.ex (hunk 3 above)
Comment (lukeledet) [reply lib/slax/slack.ex:68]: Good point, not sure why this was changed, I'll look into it

**Review by oohnoitz — DISMISSED** [review]
seems good, just had one concern.

File: lib/slax/poker/estimates.ex (hunk 2 above)
Comment (lukeledet) [reply lib/slax/poker/estimates.ex:5]: Haha that's current functionality but now that you point it out I don't think we're adding the REJECTED label when this comes up

File: lib/slax/poker.ex [hunk 4]
```diff
@@ -0,0 +1,76 @@
...
+    |> preload([:estimates])
+    |> Repo.one()
+  end
+
+  def decide(round, score) do
+    {org, repo, issue} = Github.parse_repo_org_issue(round.issue)
+
+    client = Tentacat.Client.new(%{access_token: Github.api_token()})
+
+    case Tentacat.Issues.update(client, org, repo, issue, %{labels: ["Score: #{score}"]}) do
```
Comment (grossvogel) [inline lib/slax/poker.ex:68]: I think the current label format is `points:X` ?

File: lib/slax_web/controllers/poker_controller.ex [hunk 5]
```diff
@@ -0,0 +1,169 @@
...
+        text(conn, "Something went wrong: #{inspect(other_error)}")
+    end
+  end
+
+  def start(conn, %{
+        "user_name" => user,
+        "channel_name" => channel_name,
+        "text" => "estimate " <> estimate_and_reason
+      }) do
+    with {:parse, {estimate, reason}} <- {:parse, Integer.parse(estimate_and_reason)},
```
Comment (grossvogel) [inline lib/slax_web/controllers/poker_controller.ex:51]: clever 📓

**Review by grossvogel — DISMISSED** [review]
I'm not super familiar with some of this stuff but it looks good!

**Comment by iamjoshfrank** [comment]
> except maybe next

@lukeledet - I can poll the PMs, but I don't think anyone is using / trying to use the `/poker next` command, ESPECIALLY with new kanban. There are so few instances in the real world where we're scoring Issues in consecutive numerical order.

Now, maybe in the future if we wanted "next" to mean "next in the `Estimate` column" or something like that, we might start using it again.

I'd rather see time put toward `/poker start` adding the `@doing estimate` label automatically and `/poker decide` adding the `@done estimate` label automatically, so that Issues move across the columns without clicking the manual buttons in Kanban.

**Comment by oohnoitz** [comment]
@lukeledet any reason not to merge this besides maybe a few of the things mentioned in the comments?

**Comment by lukeledet** [comment]
@oohnoitz It's been a while, I don't really remember what the hold up was. I think I just wanted to test it some more but I don't remember any big problems that would prevent it from being merged

File: lib/slax/slack.ex (hunk 3 above)
Comment (oohnoitz) [reply lib/slax/slack.ex:68]: https://api.slack.com/methods/chat.postMessage

looks like it expects a json object here, so we'd want to revert this change or just change this to:

```
Jason.encode!(%{text: message})
```

https://api.slack.com/interactivity/slash-commands#enabling-interactivity-with-slash-commands__best-practices


This change might have been made due to the way Poker shouldn't send some messages to the channel? There's a `Message Visibility` that explains it a bit more in the link above.

**Comment by oohnoitz** [comment]
gotcha. mostly curious since there are some issues with getting poker/issuebot working for repos outside of the org.

**Comment by ZachFontenot** [comment]
I tested this manually and the commands work.

**Comment by ZachFontenot** [comment]
Not sure who all to request, but we can probably get this merged, so we can move on with other things

**Review by grossvogel — APPROVED** [review]
If we think this might work, I'm all for merging it

**Pull request merged**